backend/src/services/secret/secret-queue.ts

```diff
@@ -250,6 +259,20 @@ export const secretQueueFactory = ({
         });
       }
 
+      if (recipients) {
+        await secretReminderRecipientsDAL.transaction(async (tx) => {
+          await secretReminderRecipientsDAL.delete({ secretId: newSecret.id }, tx);
+          await secretReminderRecipientsDAL.insertMany(
```

Comment:
Missing check for recipients to beg any empty array. `insertmany` would throw an error